src/RecordsModel.php

```diff
@@ -127,6 +132,22 @@ public function removeRecord(int $id): bool
         return true;
     }
 
+    // return a record by ID
+    public function returnRecord(int $id): bool
+    {
+        $query = $this->db->prepare("UPDATE `records` SET `deleted` = 0 WHERE `id` = :idNum LIMIT 1");
+
+        $query->bindParam('idNum', $id);
+
+        $query->execute();
+
+        if (!$id) {
+            return false;
+        }
+
+        return true;
```

Comment:
here you can actually just return $query->execute() and get the exact same result.

Because execute returns a bool, you're using that bool to return a bool, when you could just return the original bool :exploding\_head: